### ISSUE TITLE
Support Start/Shutdown Orders

### ISSUE THREAD

**Issue by yoshikakbudto** [bpg/terraform-provider-proxmox#392]
Running Proxmox 7.4

I use pulumi package that is using under the hood this provider's latest version to configure the Proxmox cluster. 

And i've noticed that the `startup` clause in qemu.conf either not supported or treated wrong.
To be specific: here is how the parameter `startup` looks like in qemu config file:

```
# grep "startup" /etc/pve/qemu-server/100.conf
startup: down=400
```

It was configured from a Proxmox UI here: VM/CT => Options => "Start/Shutdown Order". 

Then i do a `pulumi refresh` that should match the real cluster state with the current state file, and [got the error](https://github.com/muhlba91/pulumi-proxmoxve/issues/82) from provider:
```
* error retrieving VM: failed to decode HTTP GET response (path: nodes/pve-b2/qemu/100/config) - Reason: json: cannot unmarshal string into Go struct field GetResponseData.data.startup of type vms.CustomStartupOrder
```

**Comment by bpg** [comment]
Hey @yoshikakbudto! 👋🏼 

Thanks for the report. The startup order is not fully supported at the moment. It was added to the original PVE API client a while back, but was not actually exposed in the TF VM resource. I believe the API has changed a bit since then, and become incompatible with the client, so it throws an error you see.

**Comment by bpg** [comment]
The error was reported separately in #421, and then fixed in #428, this ticket is to implement the full startup order support in the VM resource.
